<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test</class>
 <widget class="QMainWindow" name="Test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Путь до входных файлов</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputFilePath"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Путь для выходных файлов</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="outputFilePath"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Маска входных файлов</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fileExtension"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="deleteInputFiles">
         <property name="text">
          <string>Удаление входных файлов</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>При повторении имени выходного файла</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="rewrite">
         <property name="text">
          <string>Перезаписать файл</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="createCopy">
         <property name="text">
          <string>Создать копию файла</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Режим работы программы</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="onTimer">
         <property name="text">
          <string>Работа по таймеру</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="once">
         <property name="text">
          <string>Разовый запуск</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Таймер (сек.)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="timer">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Значение 8 байт для модификации</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="bitMask"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Операция</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="operation">
         <item>
          <property name="text">
           <string>AND</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>OR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>XOR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NAND</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NOR</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>330</y>
      <width>193</width>
      <height>29</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
    </property>
   </widget>
   <widget class="QPushButton" name="stopTimer">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Остановить таймер</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Test</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
